dash/development/build_process.py

```diff
@@ -21,19 +21,20 @@ class BuildProcess(object):
     def __init__(self, main, deps_info):
         self.logger = logger
         self.main = main
+        self.build_folder = self._concat(self.main, "build")
         self.deps_info = deps_info
         self.npm_modules = self._concat(self.main, "node_modules")
         self.package_lock = self._concat(self.main, "package-lock.json")
         self.package = self._concat(self.main, "package.json")
         self._parse_package(path=self.package)
-        self.asset_paths = (self.build_folder, self.npm_modules)
+        self.asset_paths = (self.deps_folder, self.npm_modules)
 
     def _parse_package(self, path):
         with open(path, "r") as fp:
             package = json.load(fp)
             self.version = package["version"]
             self.name = package["name"]
-            self.build_folder = self._concat(self.main, os.pardir, "deps")
+            self.deps_folder = self._concat(self.main, os.pardir, "deps")
             self.deps = package["dependencies"]
```

Comment:
just so I understand this clearly. The new `self.build_folder` is located at `dash_renderer/build`, and the `dash_renderer/../deps `is now `self.deps_folder`??

Reply:
exactly - `dash/deps` is all the 3rd-party packages we load on the page, and `dash/dash-renderer/build` is the bundles we build.